### ISSUE TITLE
AP-522: Accessibility - Alt Tags/Translations

### ISSUE THREAD

**Issue by stavares843** [Satellite-im/Core-PWA#1054]
Ensure that all the components a user can navigate to work with a screen reader, and have translations

**Comment by stavares843** [comment]
> Joe McGrath commented in Jira:

It’s my understanding that decorative images should have alt attributes, but leave them blank.https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt
